model_signing/signature/key.py

```diff
@@ -95,12 +95,13 @@ def from_path(cls, key_path: str):
 
     def verify(self, bundle: bundle_pb.Bundle) -> None:
         statement = json_format.Parse(
-            bundle.dsse_envelope.payload, statement_pb.Statement())
+            bundle.dsse_envelope.payload,
+            statement_pb.Statement())  # pylint: disable=no-member
```

Comment:
```suggestion
            statement_pb.Statement()  # pylint: disable=no-member
        )
```

Reply:
This is to try and be consistent with the python style, but given that the linter didn't pick it up and we don't yet have an autoformatter, it's mostly a nit.